### PR TITLE
[16.0][OU-ADD] uom: Nothing to do

### PR DESCRIPTION
**Pull request by remytms**
New unit, added by default update mechanism. Nothing to do.

### CODE REVIEW

**Comment by remytms** [comment]
@legalsylvain can you reference this PR with `ocabot migration uom` ? :) Thanks a lot.

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -786,7 +786,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |del| transifex                                 |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| uom                                             |                      |                                                 |
+| uom                                             | Nothing to do        |No DB layout changes                                                 |
```
Comment (pedrobaeza) [inline docsource/modules150-160.rst]: Don't write `No DB layout changes` here. This is put by the module. And if you put it, remove the extra spaces for proper RST syntax.

Comment (remytms) [reply docsource/modules150-160.rst]: Well noted for the `No DB layout changes`. And thank you for spotting the spaces issue. :)

I removed the `No DB layout changes`, and fixed the spaces.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**